README.md

```diff
@@ -1 +1,68 @@
 # Envelopes
+
```

Comment:
Consider dropping the newlines after # for consistency.

Reply:
Apparently the preferred way to write markdown is with new-lines.
https://www.markdownguide.org/basic-syntax/#heading-best-practices

A lot of our docs have been written without them, but I tried to be consistent here.